assets/rln-relay/rln-relay.msc

```diff
@@ -0,0 +1,33 @@
+# MSC for some fictional process
+msc {
+  hscale = "2";
+
+  d [label = " "], a [label = " "],b [label = " "];
+  a rbox a [label="Relay Node"], 
```

Comment:
Maybe annotate Relay Node with A and B? To make them more distinct

Reply:
Actually, what is the first relay node doing? Don't see anything attached to it

Reply:
I added a second node following your other comments
>I see PR is in draft, but it would be nice to add a second Relay node in case this isn't planned already
The registration is a simple procedure and only involves one node and the contract, shall I visualize the registration of two different nodes?

Reply:
> Maybe annotate Relay Node with A and B? To make them more distinct

done